packages/core/src/components/TableFilters/stories/StoryComponent.tsx

```diff
@@ -0,0 +1,142 @@
+import { BaseSyntheticEvent, useState } from 'react';
+import { assocPath, dissocPath } from '@ssa-ui-kit/utils';
+import {
+  Popover,
+  PopoverContent,
+  PopoverDescription,
+} from '@components/Popover';
+import {
+  AccordionTitle,
+  AccordionGroup,
+  AccordionGroupContextProvider,
+} from '@components/AccordionGroup';
+import {
+  tableFilterDividerStyles,
+  tableFilterPopoverContentStyles,
+} from '../styles';
+import { mockData } from './mockData';
+import {
+  TableFilterCheckbox,
+  TableFilterTrigger,
+  TableFiltersAccordion,
+  TableFiltersAccordionContent,
+  TableFiltersButtons,
+} from '..';
+
+export type KeysNames =
+  | 'checkbox1'
+  | 'checkbox2'
+  | 'checkbox3'
+  | 'checkbox4'
+  | 'checkbox5'
+  | 'running'
+  | 'btcfdusd'
+  | 'binance';
+
+type AccordionNames = 'strategy' | 'status' | 'pairs' | 'exchange';
+
+export type InitialState = Record<
+  AccordionNames,
+  Partial<Record<KeysNames, boolean>>
+>;
+
+const initialState: InitialState = {
+  strategy: {
+    checkbox1: true,
+    checkbox4: true,
+  },
+  status: {
+    running: true,
+  },
+  pairs: {
+    ['btcfdusd']: true,
+  },
+  exchange: {
+    binance: true,
+  },
+};
+
+export const StoryComponent = () => {
+  const [checkboxData, setCheckboxData] = useState(initialState);
+  const handleCheckboxChange = (path: string[]) => (newState: boolean) => {
+    if (newState) {
+      setCheckboxData(assocPath(path, newState));
+    } else {
+      setCheckboxData(dissocPath(path));
+    }
+  };
+  const onSubmit = (event: BaseSyntheticEvent) => {
+    event.preventDefault();
+    console.log('>>>onSubmit data', checkboxData);
+  };
+  const onReset = () => {
+    setCheckboxData(() => initialState);
+  };
+  return (
+    <div
+      css={{
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}>
+      <Popover>
```

Comment:
I think, we should move this component to Widgets.
It doesn't belong to the Core sub-project.

Reply:
I mean the whole `TableFilters` directory. 
Also, this implementation should be not just a Story, but a fully-functioning exported component. So that we can easily re-use it in CTP.

Reply:
Agree, done